namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-manage-adjudications-api-prod/resources/ap-data.tf

```diff
@@ -0,0 +1,60 @@
+module "analytical-platform" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-irsa?ref=1.1.0"
+
+  namespace        = var.namespace
+  eks_cluster_name = var.eks_cluster_name
+  role_policy_arns = [aws_iam_policy.analytical-platform.arn]
+  service_account  = "${var.namespace}-analytical-platform"
+  # NB: service account name must be unique within Cloud Platform (IAM role name is derived from it)
+}
+
+resource "aws_iam_policy" "analytical-platform" {
+  name   = "${var.namespace}-analytical-platform"
+  policy = data.aws_iam_policy_document.analytical-platform.json
+  # NB: IAM policy name must be unique within Cloud Platform
+
+  tags = {
+    business-unit          = var.business_unit
+    team_name              = var.team_name
+    application            = var.application
+    is-production          = var.is_production
+    namespace              = var.namespace
+    environment-name       = var.environment-name
+    owner                  = var.team_name
+    infrastructure-support = var.infrastructure_support
+  }
+}
+
+data "aws_iam_policy_document" "analytical-platform" {
+  statement {
+    actions = [
+      "s3:ListBucket",
+    ]
+    resources = [
+      "arn:aws:s3:::mojap-adjudications-insights",
```

Comment:
Assuming this based on https://github.com/moj-analytical-services/data-engineering-exports/pull/35/files#diff-e650733f77039411893be8e298fc57db6f49c864ff3d4b281326d699637fd383R1

For Incentives we have https://github.com/moj-analytical-services/data-engineering-exports/blob/496b79bace7ea5efe294a787a3909140648e4d27/pull_datasets/incentives.yaml#L1 which created the bucket `mojap-incentives`